exercises/concept/vehicle-purchase/.meta/design.md

```diff
@@ -0,0 +1,29 @@
+
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student about conditionals in R.
+
+## Learning objectives
+
+- Know that the relational operators include `==`, `!=`, `>=`, `<=`.
+- Know the syntax of a block if-else.
+- Know about the `ifelse()` function.
```

Comment:
If use of `ifelse()` with vectors is out of scope for this exercise (which I agree with), then I don't agree with "knowing about the `ifelse()` function" being one of the learning objectives.

Reply:
changed